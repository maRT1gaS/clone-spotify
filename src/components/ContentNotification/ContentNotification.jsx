import React from 'react';
import PropTypes from 'prop-types';
import styles from './ContentNotification.module.css';

export function ContentNotification({ title }) {
  return (
    <div className={styles.container}>
      <h2 className={styles.title}>{title}</h2>
    </div>
  );
}

ContentNotification.propTypes = {
  title: PropTypes.string.isRequired,
};
