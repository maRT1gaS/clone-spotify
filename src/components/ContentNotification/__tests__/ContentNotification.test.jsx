import React from 'react';
import { shallow } from 'enzyme';
import { ContentNotification } from '../ContentNotification';

describe('ContentNotification', () => {
  it('should render correctly', () => {
    const component = shallow(
      <ContentNotification title='В данный момент данных нет!' />
    );
    expect(component).toMatchSnapshot();
  });
});
