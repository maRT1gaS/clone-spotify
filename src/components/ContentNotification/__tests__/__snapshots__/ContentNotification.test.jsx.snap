// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentNotification should render correctly 1`] = `
<div
  className="container"
>
  <h2
    className="title"
  >
    В данный момент данных нет!
  </h2>
</div>
`;
